src/main/java/cinema/controllers/MovieSessionController.java

```diff
@@ -0,0 +1,43 @@
+package cinema.controllers;
+
+import cinema.model.dto.request.MovieSessionRequestDto;
+import cinema.model.dto.response.MovieSessionResponseDto;
+import cinema.model.mapper.MovieSessionMapper;
+import cinema.service.MovieSessionService;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/movie-sessions")
+public class MovieSessionController {
+    private final MovieSessionMapper movieSessionMapper;
+    private final MovieSessionService movieSessionService;
+
+    public MovieSessionController(MovieSessionMapper movieSessionMapper,
+                                  MovieSessionService movieSessionService) {
+        this.movieSessionMapper = movieSessionMapper;
+        this.movieSessionService = movieSessionService;
+    }
+
+    @PostMapping
+    public void addMovieSession(@RequestBody MovieSessionRequestDto movieSessionRequestDto) {
+        movieSessionService.add(movieSessionMapper.toModel(movieSessionRequestDto));
+    }
+
+    @GetMapping
+    public List<MovieSessionResponseDto> getAllMovieSessions(
+            @RequestParam Long movieId,
+            @RequestParam("date") @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) LocalDate date) {
```

Comment:
```suggestion
            @RequestParam @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) LocalDate date) {
```
цього не потрібно, якщо в тебе ідентичні назви